tests/main_sa/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2020, RTE (http://www.rte-france.com)
+# See AUTHORS.txt
+# All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+#
+
+add_test(
+    NAME MAIN_SA.varEnv
+    COMMAND DynaFlowLauncher --network=res/TestIIDM_launch_infinite.iidm --config=res/launch_infinite.json --contingencies=res/contingencies.json
+    WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+  )
+# Test will fail because environement variables are not set
+set_tests_properties(MAIN_SA.varEnv PROPERTIES WILL_FAIL TRUE)
+
+macro(DEFINE_LAUNCH_TEST_SA _name)
+  add_test(
+    # Use some per-module/project prefix so that it is easier to run only tests for this module
+    NAME MAIN_SA.${_name}
+    COMMAND ${CMAKE_COMMAND} -DEXE=$<TARGET_FILE:DynaFlowLauncher> -DTEST_NAME=${_name} -DDIFF_SCRIPT=${CMAKE_SOURCE_DIR}/scripts/diffContingencies.py -DCHECK_SCRIPT=${CMAKE_SOURCE_DIR}/scripts/checkContingencies.py -P ${CMAKE_SOURCE_DIR}/cmake/TestMainSA.cmake
+    WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+  )
+  set_property(TEST MAIN_SA.${_name} PROPERTY ENVIRONMENT IIDM_XML_XSD_PATH=${DYNAWO_HOME}/share/iidm/xsd/)
+  set_property(TEST MAIN_SA.${_name} APPEND PROPERTY ENVIRONMENT LD_LIBRARY_PATH=${DYNAWO_HOME}/lib:${LD_LIBRARY_PATH})
```

Comment:
Add dynawo algorithms path here to mahe unit tests work (same for other macro)

Reply:
Done